1-js/13-modules/02-import-export/article.md

```diff
@@ -306,133 +305,152 @@ import func from '/path/to/func.js';
+    ```js
+    export * from './user.js'; // para reexportar exports com nome
+    export {default} from './user.js'; // para reexportar o export default
+    ```
+
+Essas esquisitices de reexportar o export default são um dos motivos pelos quais alguns desenvolvedores não gostam deles.
+
+## Resumo
+
+Aqui estão todos os tipos de `export` que abordamos neste e em artigos anteriores.
+
+Você pode verificar lendo-os e relembrando o que eles significam:
+
+- Antes da declaração de uma class/function/..:
   - `export [default] class/function/variable ...`
 - Standalone:
   - `export {x [as y], ...}`.
-- Re-export:
-  - `export {x [as y], ...} from "mod"`
-  - `export * from "mod"` (doesn't re-export default).
-  - `export {default [as y]} from "mod"` (re-export default).
+- Reexportação:
+  - `export {x [as y], ...} from "module"`
+  - `export * from "module"` (não reexporta o default).
+  - `export {default [as y]} from "module"` (reexporta o default).
 
 Import:
 
-- Named exports from module:
-  - `import {x [as y], ...} from "mod"`
-- Default export:  
-  - `import x from "mod"`
-  - `import {default as x} from "mod"`
-- Everything:
-  - `import * as obj from "mod"`
-- Only fetch/evalute the module, don't import:
-  - `import "mod"`
+- Exports com nome do módulo:
+  - `import {x [as y], ...} from "module"`
+- Export default:  
+  - `import x from "module"`
+  - `import {default as x} from "module"`
+- Tudo:
+  - `import * as obj from "module"`
+- Importar o módulo (seu código executado), sem o atribuir a uma variável:
+  - `import "module"`
 
-We can put import/export statements below or after other code, that doesn't matter.
+Podemos colocar as declarações `import/export` no início ou no final de um script, isso não importa.
 
-So this is technically fine:
+Então, tecnicamente, esse código é correto:
 ```js
 sayHi();
 
-import {sayHi} from './say.js'; // import at the end of the file
+// ...
+
+import {sayHi} from './say.js'; // import no final do arquivo
 ```
 
-In practice imports are usually at the start of the file, but that's only for better convenience.
+Na prática, as importações normalmente ficam no início do arquivo, mas isso é apenas por conveniência.
 
-**Please note that import/export statements don't work if inside `{...}`.**
+**Observe que as declarações de importação e exportação não funcionam dentro de `{...}`.**
 
-A conditional import, like this, won't work:
+Um import condicional como esse não vai funcionar:
 ```js
 if (something) {
-  import {sayHi} from "./say.js"; // Error: import must be at top level
+  import {sayHi} from "./say.js"; // Erro: importação deve estar em um nível acima
 }
 ```
 
-...But what if we really need to import something conditionally? Or at the right time? Like, load a module upon request, when it's really needed?
+... Mas e se realmente precisarmos importar algo condicionalmente? Ou na hora certa? Como carregar um módulo mediante solicitação, quando é realmente necessário?
```

Comment:
I think you missed a comma after "Como". With a comma, the following becomes an example rather than a question How.